crates/cxx-qt-gen/src/generator/cpp/qobject.rs

```diff
@@ -99,6 +100,7 @@ impl GeneratedCppQObject {
     pub fn from(
         structured_qobject: &StructuredQObject,
         type_names: &TypeNames,
+        opt: &GeneratedOpt,
```

Comment:
This doesn't check the `#[cfg()]` attributes on the QObject itself, right?

So it would still not be possible to conditionally enable/disable an entire QObject...

I should also note that we would likely need to update the structuring phase to filter out any disabled QObjects/QEnums...

Reply:
Wonder how hard QObject actually is, and ultimately if you have a qenum without the cfg but the qobject with the cfg then i think that is your problem. As same would happen if you have a struct vs impl.

Reply:
(whether our generation throws things out i'm not sure if you are missing a qobject for a signal/method/enum/inherit etc)

Reply:
Done